Source/WebKit/UIProcess/ios/WKActionSheetAssistant.mm

```diff
@@ -396,6 +396,9 @@ - (void)showImageSheet
             targetURL = alternateURL;
         auto elementBounds = _positionInformation->bounds;
         auto elementInfo = adoptNS([[_WKActivatedElementInfo alloc] _initWithType:_WKActivatedElementTypeImage URL:targetURL imageURL:imageURL location:_positionInformation->request.point title:_positionInformation->title ID:_positionInformation->idAttribute rect:elementBounds image:_positionInformation->image.get() imageMIMEType:_positionInformation->imageMIMEType userInfo:userInfo]);
+        if (!targetURL && alternateURL)
```

Comment:
I realized this check will never succeed, since `targetURL` always gets set.

Basically, we only want this code to run when the `targetURL` is the same as the `alternateURL`, so we need an equality check.

Reply:
@pxlcoder should it be `targetURL && [targetURL isEqual:alternateURL]` or just `[targetURL isEqual:alternateURL]`?

Since technically, targetURL could be null at this point

Reply:
Just `[targetURL isEqual:alternateURL]`. Sending a message to `nil` will return 0.